todocore/src/main/resources/data.sql

```diff
@@ -0,0 +1,210 @@
+CREATE TABLE armor
```

Comment:
You shouldn't commit this file, it's not related to the task, it should be your local file